VirtualH89/Src/AddressBus.cpp

```diff
@@ -10,10 +10,11 @@
 #include "MemoryDecoder.h"
 #include "InterruptController.h"
 
-AddressBus::AddressBus(InterruptController* ic): ic_m(ic), mem(NULL)
+AddressBus::AddressBus(InterruptController* ic):
```

Comment:
Did uncrustify make these changes to AddressBus, without any other changes made to the file?